splink/linker.py

```diff
@@ -2153,6 +2153,7 @@ def truth_space_table_from_labels_column(
         labels_column_name,
         threshold_actual=0.5,
         match_weight_round_to_nearest: float = None,
+        positives_not_captured_by_blocking_rules_scored_as_zero: bool = True,
```

Comment:
Allow the user to control this.  

In some cases the user may be interested in how good their scoring model is, or to analyse the difference blocking is making (what proportion of the false positives and negatives are due to bad blocking, and what proportion are due to bad scoring)